programs/zerocal.cpp

```diff
@@ -220,7 +231,7 @@ class AdjustJointStep : public CalibrationStep {
 					LowLevelWam<DOF>& llw = wam.getLowLevelWam();
 
 					// Record actual joint position, not commanded joint position
-					zeroPos[j] = wam.getJointPositions()[j];
+					zeroPos[j] = wam.getJointPositions()[j] - calPos[j];
```

Comment:
We should add a comment that this change is actually from the `zerocal` branch upstream.